### PR TITLE
This is not needed.

### PR DESCRIPTION
**Pull request by Jint3x-experiment**
Remove this license part.

### CODE REVIEW

**Comment by Jint3x** [comment]
Are you sure?

**Comment by Jint3x-experiment** [comment]
Yea, I'm pretty sure it won't be needed.

File: LICENSE [hunk 1]
```diff
@@ -12,13 +12,9 @@ furnished to do so, subject to the following conditions:
 The above copyright notice and this permission notice shall be included in all
 copies or substantial portions of the Software.
```
Comment (Jint3x) [inline LICENSE:13]: Remove this too?

**Review by Jint3x — CHANGES_REQUESTED** [review]
Please check the comments I've left for you and update them.

**Review by Jint3x — APPROVED** [review]
LGTM!

**Pull request merged**